presto-elasticsearch/src/main/java/com/facebook/presto/elasticsearch/ElasticsearchQueryBuilder.java

```diff
@@ -109,6 +110,10 @@ public SearchRequestBuilder buildScrollSearchRequest()
                 .setFetchSource(fields.toArray(new String[0]), null)
                 .setQuery(buildSearchQuery())
                 .setPreference("_shards:" + shard)
+                // Scroll requests have optimizations that make them faster when the sort order is _doc.
```

Comment:
how about move the comments above line 106, so that comments not interleave with code?

Reply:
done